src/IIIFPresentation/API.Tests/Integration/ModifyManifestUpdateTests.cs

```diff
@@ -151,12 +151,34 @@ public async Task PutFlatId_Update_BadRequest_IfParentAndSlugAlreadyExist_ForCol
         // Assert
         response.StatusCode.Should().Be(HttpStatusCode.Conflict);
     }
+
+    [Fact]
+    public async Task PutFlatId_Update_BadRequest_IfParentAndSlug_VaryCase_ForCollection()
```

Comment:
Tiny thing but `_BadRequest_` for these 2 tests should be `_Conflict_`

Reply:
More than 2 actually, checked all over and fixed naming